saltgui/static/scripts/output.js

```diff
@@ -21,6 +21,63 @@
 
 class Output {
 
+  // format a JSON object
+  // based on an initial indentation and an indentation increment
+  static _formatJSON0(value, indentStr, indentCnt) {
+
+    if(typeof value !== "object") {
+      // a simple type
+      // leave that to the builtin function
+      return JSON.stringify(value);
```

Comment:
Add the indentation here from the `indent` parameter

Reply:
the indentation is only added for the content of arrays/objects.
otherwise the value `123` in `{"a":123}` would also be indented.
strings, numbers, the starting `[` and the starting `{` are not indented.